web/webpack.config.dev.js

```diff
@@ -35,13 +35,18 @@ const config = {
           'postcss-loader',
           {
             loader: 'sass-loader',
-            options: { includePaths: [path.resolve(__dirname, 'node_modules')] }
+            options: {
+              sassOptions: {
+                includePaths: [path.resolve(__dirname, 'node_modules')]
+              }
+            }
           }
         ]
       },
       {
         test: /\.(woff2?|ttf|otf|eot|svg)$/,
-        loader: 'file-loader'
+        loader: 'file-loader',
+        options: { esModule: false }
```

Comment:
I'm curious about this.

Reply:
Oops, forgot to roll that back. I was trying to get the `file-loader` update to work, but it wouldn't. So there's no reason for this now.